eox_tenant/edxapp_wrapper/backends/edx_auth_i_v1.py

```diff
@@ -1,5 +1,6 @@
 """ Backend abstraction. """
-from openedx.core.djangoapps.oauth_dispatch.dot_overrides.backends import EdxRateLimitedAllowAllUsersModelBackend  # pylint: disable=import-error
+# pylint: disable=import-error
```

Comment:
With this change you are disabling all the import errors, and we don't want to do that, if this line is too long my suggestion is the following
``` 
from openedx.core.djangoapps.oauth_dispatch.dot_overrides.backends import (  # pylint: disable=import-error
    EdxRateLimitedAllowAllUsersModelBackend,
)
```

Reply:
Thank you! I'll change it